### DISCUSSION TITLE
Frameworks / Libraries

### DISCUSSION THREAD

**Discussion by l7f19** [felixhoffmnn/aswe#3] [General]
## Speech Recognition

Aus der Aufgabe geht hervor, dass die Interaktion mit dem späteren Interface / API mittels **Sprachsteuerung** erfolgen soll. 

- https://pypi.org/project/SpeechRecognition/
- https://pypi.org/project/pyttsx3/

**Comment by felixhoffmnn** [comment]
Google bietet ebenfalls eine **speech-to-text API**. Für die Nutzung erhält man ein **Kontingent von 60 Minuten** frei zur Verfügung.

- https://cloud.google.com/speech-to-text/

**Comment by l7f19** [comment]
[Library für VVS API access](https://github.com/zaanposni/vvspy)
